tests/wasm/math.rs

```diff
@@ -4,6 +4,20 @@ use wasm_bindgen_test::*;
 #[wasm_bindgen(module = "tests/wasm/math.js")]
 extern "C" {
     fn js_auto_bind_math();
+
+    #[wasm_bindgen(js_name = roundtrip)]
```

Comment:
I think it's a little subtle why these use `f64`, so a comment explaining it would be nice.